desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraph.java

```diff
@@ -278,9 +284,31 @@ public void paint(Graphics g) {
                             + "So add one or more such components.";
                     imagePath = "images/window/canvas-bg-plus.png";
                 } else {
-                    title = null;
+                    title=null;
                     subTitle = null;
                     imagePath = null;
+                    
+                    try {
+                        if (!_analysisJobBuilder.isConfigured(true)) {
+                            subTitle="Job is not correctly configured";
+                            warning=true; 
+                        }
+                    } catch (Exception ex) {
+                        logger.debug("Job not correctly configured", ex);
+                        final String errorMessage;
+                        if (ex instanceof UnconfiguredConfiguredPropertyException) {
+                            UnconfiguredConfiguredPropertyException unconfiguredConfiguredPropertyException = (UnconfiguredConfiguredPropertyException) ex;
+                            ConfiguredPropertyDescriptor configuredProperty = unconfiguredConfiguredPropertyException
+                                    .getConfiguredProperty();
+                            ComponentBuilder componentBuilder = unconfiguredConfiguredPropertyException.getComponentBuilder();
+                            errorMessage = "Property '" + configuredProperty.getName() + "' in "
+                                    + LabelUtils.getLabel(componentBuilder) + " is not set!";
+                        } else {
+                            errorMessage = ex.getMessage();
+                        }
+                        subTitle="Job error status: " + errorMessage;
```

Comment:
This is really what I mean by "robot language". Don't have the "Job error status: " prefix here. Just keep the error message (as subtitle) and make sure the main title is human language.